code/modules/mob/living/carbon/human/human.dm

```diff
@@ -207,8 +207,11 @@
 // this could be made more general, but for now just handle mulebot
 /mob/living/carbon/human/Crossed(atom/movable/AM)
 	var/mob/living/simple_animal/bot/mulebot/MB = AM
+	var/obj/vehicle/sealed/car/C = AM
 	if(istype(MB))
 		MB.RunOver(src)
+	if(istype(C))

```

Comment:
Is there not a `Crossed()` component signal?

At least make this an else if.

Reply:
Done. I made this pr about two and a half weeks before i finally figured out components